doc/ale.txt

```diff
@@ -393,41 +407,40 @@ g:ale_fortran_gcc_options                           *g:ale_fortran_gcc_options*
 
 
 -------------------------------------------------------------------------------
-4.7. html-tidy                                   *ale-linter-options-html-tidy*
+4.7. fortran-gcc                               *ale-linter-options-fortran-gcc*
 
-g:ale_html_tidy_executable                         *g:ale_html_tidy_executable*
+g:ale_fortran_gcc_options                           *g:ale_fortran_gcc_options*
 
   Type: |String|
-  Default: `'tidy'`
-
-  This variable can be changed to change the path to tidy.
+  Default: `'-Wall'`
 
+  This variable can be changed to modify flags given to gcc.
 
-g:ale_html_tidy_args                                     *g:ale_html_tidy_args*
 
-  Type: |String|
-  Default: `'-q -e -language en'`
+-------------------------------------------------------------------------------
+4.8. shell                                           *ale-linter-options-shell*
 
-  This variable can be changed to change the arguments provided to the
-  executable.
+g:ale_linters_sh_shell_default_shell     *g:ale_linters_sh_shell_default_shell*
 
-  ALE will attempt to automatically detect the appropriate file encoding to
-  provide to html-tidy, and fall back to UTF-8 when encoding detection fails.
+  Type: |String|
+  Default: The current shell (`$SHELL`) or `'bash'` if that cannot be read.
 
-  The recognized file encodings are as follows: ascii, big5, cp1252 (win1252),
-  cp850 (ibm858), cp932 (shiftjis), iso-2022-jp (iso-2022), latin1, macroman
-  (mac), sjis (shiftjis), utf-16le, utf-16, utf-8
+  When ALE runs the linter for shells with the `-n` flag, it will attempt to
+  read the shell from the shebang (`#!`) line from the shell script to
+  determine the shell program to run. When this detection fails, this variable
+  will be used instead.
 
 
 -------------------------------------------------------------------------------
-4.8. jshint                                         *ale-linter-options-jshint*
+4.9. shellchek                                  *ale-linter-options-shellcheck*
 
-g:ale_javascript_jshint_executable         *g:ale_javascript_jshint_executable*
+g:ale_linters_sh_shellckeck_exclusions *g:ale_linters_sh_shellckeck_exclusions*
 
   Type: |String|
-  Default: `'jshint'`
+  Default: `''`
+
+  Set this variable to exclude test for shellcheck (-e option);
```

Comment:
Woopsie the semi-column, too much c/cpp latetly.
For the -e as I'm not a shellcheck user I've report to the doc and I don't think it's suppose to be plural : https://github.com/koalaman/shellcheck/wiki/Ignore
I'll dig through that a bit more.